### PR TITLE
Calculated properties

### PR DESCRIPTION
**Pull request by ScottKwang**
- changed `test_review_view_get` to `test_team_view_get` so that this test does not start to fail as we add more serialized fields in the following questions
- add `test_review_view_get_with_normalized_score`
- add `test_game_view_get_with_calculated_values`
- update `test_game_view_get_with_filter` to include the `average_score`
- update `test_review_view_get_with_sort` to include the `normalized_score`

- updated all tests to include `id` in the output

### CODE REVIEW

File: kettocritic/serializers.py [hunk 1]
```diff
@@ -27,16 +27,19 @@ def get_serialized_model(self, instance):
...
 
 
 class ReviewerSerializer(ModelSerializer):
-    fields = ['name', 'team']
+    fields = ['id', 'name', 'team']
 
 
 class ReviewSerializer(ModelSerializer):
-    fields = ['game', 'reviewer', 'score', 'title']
+    fields = ['id', 'description', 'created_on', 'game', 'reviewer', 'score', 'title']
```
Comment (ScottKwang) [inline kettocritic/serializers.py]: maybe we should just remove `description` to lower the clutter in the tests and output

File: kettocritic/models.py [hunk 2]
```diff
@@ -32,14 +32,18 @@ class Team(BaseModel):
     name = CharField(unique=True)
     website = CharField(unique=True)
 
+    @property
```
Comment (ScottKwang) [inline kettocritic/models.py]: example of model @property

File: kettocritic/serializers.py [hunk 3]
```diff
@@ -27,16 +27,19 @@ def get_serialized_model(self, instance):
...
 class GameSerializer(ModelSerializer):
-    fields = ['name']
+    fields = ['id', 'name']
 
 
 class TeamSerializer(ModelSerializer):
-    fields = ['name', 'website']
+    fields = ['id', 'name', 'website', 'num_reviews', 'num_reviewers']
+
+    def get_num_reviews(self, instance):
```
Comment (ScottKwang) [inline kettocritic/serializers.py]: example serializer property

File: kettocritic/serializers.py [hunk 4]
```diff
@@ -27,16 +27,19 @@ def get_serialized_model(self, instance):
 
 
 class GameSerializer(ModelSerializer):
-    fields = ['name']
+    fields = ['id', 'name']
 
 
 class TeamSerializer(ModelSerializer):
-    fields = ['name', 'website']
+    fields = ['id', 'num_reviews', 'num_reviewers']
```
Comment (mroyce) [inline kettocritic/serializers.py]: does the serializer automatically grab the `num_reveiwers` property from the model or do we have to create a method for it?  wondering if we should just define the logic for the calculated property in the model and and grab it there.

File: test.py [hunk 5]
```diff
@@ -69,24 +98,24 @@ def test_review_view_get_with_sort(self):
...
+            {'description': 'Dwyane Wade Takes an in-depth look at Cuphead', 'reviewer': 10, 'created_on': '2017-10-29 21:15:16.529267', 'id': 9, 'normalized_score': 60, 'score': 9, 'game': 3, 'title': "Dwyane Wade's Review of Cuphead"},
+            {'description': 'LeBron James Takes an in-depth look at Destiny 2', 'reviewer': 8, 'created_on': '2017-10-29 21:15:16.529267', 'id': 12, 'normalized_score': 40, 'score': 12, 'game': 4, 'title': "LeBron James's Review of Destiny 2"},
+            {'description': 'Derrick Rose Takes an in-depth look at Super Smash Bros. for Wii U', 'reviewer': 9, 'created_on': '2017-10-29 21:15:16.529267', 'id': 15, 'normalized_score': 100, 'score': 15, 'game': 5, 'title': "Derrick Rose's Review of Super Smash Bros. for Wii U"},
+            {'description': 'Dwyane Wade Takes an in-depth look at Super Smash Bros. for Wii U', 'reviewer': 10, 'created_on': '2017-10-29 21:15:16.529267', 'id': 18, 'normalized_score': 80, 'score': 18, 'game': 5, 'title': "Dwyane Wade's Review of Super Smash Bros. for Wii U"}
```
Comment (mroyce) [inline test.py]: reeeeeeeeeee

**Review by mroyce — APPROVED** [review]

File: kettocritic/serializers.py [hunk 6]
```diff
@@ -27,16 +27,19 @@ def get_serialized_model(self, instance):
 
 
 class GameSerializer(ModelSerializer):
-    fields = ['name']
+    fields = ['id', 'name']
 
 
 class TeamSerializer(ModelSerializer):
-    fields = ['name', 'website']
+    fields = ['id', 'name', 'website', 'num_reviews', 'num_reviewers']
```
[the remark this answers is not in the document]
Comment (ScottKwang) [reply kettocritic/serializers.py]: errors

File: kettocritic/serializers.py (hunk 4 above)
Comment (ScottKwang) [reply kettocritic/serializers.py]: i think we need to define the logic for getting it from the model property

**Pull request merged**